prime-router/src/main/resources/metadata/hl7_mapping/resources/Observation/OBXValue.yml

```diff
@@ -69,10 +69,10 @@ elements:
 
   - name: obx-value-ce
     condition: '%context.extension(%`rsext-obx-observation`).extension.where(url = "OBX.2").value = "CE"'
-    schema: classpath:/metadata/hl7_mapping/datatypes/codeableConcept/CE.yml
+    schema: classpath:/metadata/hl7_mapping/datatypes/codeableConcept/CWE.yml
```

Comment:
Nit: we could collapse `obx-value-ce` and `obx-value-cwe` into a singular `obx-value` I think since they're now functionally the same.

Reply:
Ah great idea, fixed this: https://github.com/CDCgov/prime-reportstream/pull/14724/commits/936a0e9cbc4d7b8e8acb9e661f0b087f4aac6484

Side note: At first I thought you meant to just delete it. After I did that the e2e tests were still passing apparently because our data comparison is very forgiving if data is missing so long as the structure is the same. I've pushed up a few minor changes there too.